src/transformers/models/big_bird/modeling_big_bird.py

```diff
@@ -568,7 +568,13 @@ def bigbird_block_sparse_attention(
         if from_seq_len in [1024, 3072, 4096]:  # old plans used in paper
             rand_attn = [
                 self._bigbird_block_rand_mask(
-                    self.max_seqlen, self.max_seqlen, from_block_size, to_block_size, n_rand_blocks, last_idx=1024
+                    self.max_seqlen,
+                    self.max_seqlen,
+                    from_block_size,
+                    to_block_size,
+                    n_rand_blocks,
+                    self.training,
```

Comment:
Looks like we set `deterministic=self.training`. Don't think this is quite right, should be the other way round, e.g.
```suggestion
                    not self.training,
```